src/UI/Component/Floatable/Factory.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/* Copyright (c) 2017 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Floatable;
+
+use ILIAS\UI\Component\Component;
+/**
+ * Floatable factory
+ */
+interface Factory {
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       Instructional Overlays keep didactical instructions available to users
+	 *       while they carry out workflows presented by this instruction that spread
+	 *       over several screens. Instructional information and actions are made
+	 *       available at the very screen the user is working on.
+	 *   composition: >
+	 *       The Instructional Overlay is comprised of a title and a glyphed
+	 *       close-button, an assignment description (as shown in the info-tab of e.g. the
+	 *       exercise) namely textual work instructions, accompanying files as links or
+	 *       inline media, schedule information if applicable and some
+	 *       submission-Button to carry out action the single most important action of
+	 *       the workflow.
```

Comment:
This is not about the help or about operating ILIAS but about pedagogical instructions that come with quite a bit of input. It is very difficult for users to shuttle back and forth between the assignment and their object they work on to complete the assignment. This supposedly helps them to focus on completing their portfolio in line with the faculty requirements, deadlines and formal requirements